docs/sending-tx/en/sending-tx-ultimate-guide.md

```diff
@@ -0,0 +1,199 @@
+
+### Rejected by RPC
+The sending transaction is directly rejected by RPC. The reason for rejection can be found in the `message` field of RPC Response.
+
+1. `Invalid parameters: tx', 'data': '"Failed imported to deferred pool: Tx with same nonce already inserted. To replace it, you need to specify a gas price > 5` The error code indicates that the same nonce has been used in the transaction pool and cannot be reused. However, you can replace the existed transaction by increasing the gasPrice.
+2. `'Invalid parameters: tx', 'data': '"Transaction 0x2004b0aea956e8cfad601cd6daad5630c1a95624bad446d1966895973325136c is discarded due to a too stale nonce` This indicates that the same nonce used has been used in history and cannot be used again.
+3. `Sending transactions to invalid address. The first four bits must be 0x0 (built-in/reserved), 0x1 (user-account), or 0x8 (contract).` This indicates that the receiving address of the transaction is wrong.
+4. `Transaction {:?} is discarded due to in too distant future` This indicates that the transaction is using a nonce that is too large for the account's current nonce
+5. `tx already exist` This indicates that the transaction already exists.
+
+Sometimes, the `data` of Response will also contain some error information. （This field is a hex-encoded string. The parsing method is: hex -> buffer -> UTF8 string）
+
+### Stucked in the transaction pool
+The transaction is sent successfully but not be packaged. There are two possible reasons:
+
+1. The skipped nonce is used, and the transaction of the previous nonce has not been generated or executed.
+2. The account sending this transaction does not have enough balance to pay the transaction.
+
+### Executed failed
+Execution failure is usually due to an error occurred during the execution process of the contract, which then caused the transaction failure. Such errors are mostly caused by contract execution failures or errors returned when estimating gas cost through the estimate interface.
+You can check the specific reason for the transaction failure in the `txExecErrorMsg` under receipt:
+
+1. `VmError(OutOfGas)` The transaction specified gas fee is not enough
+2. `VmError(ExceedStorageLimit)` The transaction specified upper-limit storage is not enough
+3. `NotEnoughCash` Insufficient user balance
+4. `Vm reverted, Reason provided by the contract: xxxx` The contract execution failed with details provided. 
+5. `VmError(BadInstruction` Contract deployment failed
+6. `Vm reverted, ` The contract execution failed with no details provided.
+
+
+## Difference between Conflux and Ethereum
+Compared to Ethereum, transactions through Conflux have three more fields: `chainId`, `storageLimit`, and `epochNumber`. The rlp encoding assembly method of the transaction is different. In addition, the v value signed by ecdsaSign will not be specifically modified in Conflux; while In Ethereum, there will be some special treatments to the v value.
```

Comment:
Maybe add:

*"Finally, while Ethereum nodes directly reject transactions if the sender has insufficient balance, Conflux nodes accept such transactions but only pack them info blocks once the user's balance can cover it."*